### PR TITLE
Add subscriptions as a property on accounts

### PR DESCRIPTION
**Pull request by IsaacMurtagh**

### CODE REVIEW

File: src/api/accounts/accounts.md [hunk 1]
```diff
@@ -32,17 +32,23 @@ individual account.
...
-| modifiedAt | {% dt Timestamp %} | When the Account was updated.                               |
-| createdBy  | {% dt CRN %}       | The User or API Key that created the Account.               |
-| modifiedBy | {% dt CRN %}       | The User or API Key that updated the Account.               |
+|     Field     |        Type        |                         Description                         |
+| :------------ | :----------------- | :---------------------------------------------------------- |
+| id            | String             | The unique identifier.                                      |
+| type          | String             | Account type, must be either 'org' or 'individual'.         |
+| name          | String             | The display name of the Account.                            |
+| region        | String             | The region that the Account will operate in.                |
+| test          | Boolean            | A flag which is only present if the account is for testing. |
```
Comment (MeganSteenkamp) [inline src/api/accounts/accounts.md:41]: I think Account here should have a capital A like it previously did, as it is a resource.

File: src/api/accounts/accounts.md [hunk 2]
```diff
@@ -32,17 +32,23 @@ individual account.
...
+| createdAt     | {% dt Timestamp %} | When the Account was created.                               |
+| modifiedAt    | {% dt Timestamp %} | When the Account was updated.                               |
+| createdBy     | {% dt CRN %}       | The User or API Key that created the Account.               |
+| modifiedBy    | {% dt CRN %}       | The User or API Key that updated the Account.               |
+| subscriptions | Array              | A list of [Subscriptions](#subscription) on the Account.    |
+
+### Subscription
```
Comment (MeganSteenkamp) [inline src/api/accounts/accounts.md:48]: You need a newline here for the table to format properly :)

File: src/api/accounts/accounts.md (hunk 1 above)
Comment (IsaacMurtagh) [reply src/api/accounts/accounts.md]: Yep, got removed when resolving a merge conflict.

File: src/api/accounts/accounts.md (hunk 2 above)
Comment (IsaacMurtagh) [reply src/api/accounts/accounts.md:48]: Thanks 🙂

**Review by MeganSteenkamp — APPROVED** [review]

**Pull request merged**